scalafix-rules/src/main/scala-2/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -176,17 +176,16 @@ final class ExplicitResultTypes(
     def qualifyingImplicit: Boolean =
       isImplicit && !isFinalLiteralVal
 
-    def matchesMemberKindAndVisibility: Boolean =
-      matchesMemberKind && matchesMemberVisibility
+    def matchesConfig: Boolean =
+      matchesMemberKind && matchesMemberVisibility && !matchesSimpleDefinition
```

Comment:
looking at this line, wouldn't it make sense to have a `&& matchesSkipSimpleDefinition` predicate instead, with the not (!) inside of the predicate itself? 
At least to me it would look more readable, without having to worry on the content of the predicate

Reply:
I personally like `!matchesSimpleDefinition` better - the "matches skip" is giving me headaches :sweat_smile: